content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -0,0 +1,183 @@
+You can export all of your resources or a subset of them based on a list of resource types.
+The `dump` command supports exporting in `wrapped-json` and `yaml`.
+
+{{% notice note %}}
+**NOTE**: Passwords are not included when exporting users.
+You must add the `password` attribute to any exported user resources before you can use them with `sensuctl create`.
+{{% /notice %}}
+
+For example, to export all resources to a file named `my-resources.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump all --format yaml --file my-resources.yaml
+{{< /code >}}
+
+To export only checks to STDOUT in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump check --format yaml
+{{< /code >}}
+
+To export only handlers and filters to a file named `my-handlers-and-filters.yaml` in `yaml` format:
+
+{{< code shell >}}
+sensuctl dump handler,filter --format yaml --file my-handlers-and-filters.yaml
+{{< /code >}}
+
+## Back up your resources
+
+Use `sensuctl dump` to back up your Sensu resources so you can [restore][3] them later with [`sensuctl create`][1].
+For example, we recommend creating a backup before you [upgrade][4] to a new version of Sensu.
+
+{{% notice note %}}
+**NOTE**: You cannot export `secrets/v1.provider` resources &mdash; make sure to omit them from your `sensuctl dump` commands.
+{{% /notice %}}
+
+Here's the step-by-step process:
+
+1. Create a backup folder.
+
+   {{< code shell >}}
+mkdir backup
+{{< /code >}}
+   
+2. Back up your pipeline resources, stripping namespaces to maintain portability and facilitate reuse.
+   
+   {{< code shell >}}
+sensuctl dump assets,checks,hooks,filters,mutators,handlers,silenced,secrets/v1.Secret \
+--format yaml | grep -v "^\s*namespace:" > backup/pipelines.yaml
+{{< /code >}}
+   
+3. Create a namespaced backup of the entire cluster (except entities, events, and apikeys).
+   
+   {{< code shell >}}
+sensuctl dump all \
+--omit entities,events,apikeys \
+--format yaml > backup/config.yaml
+{{< /code >}}
+   
+4. Back up your [role-based access control (RBAC)][2] resources.
+   
+   {{< code shell >}}
+sensuctl dump apikeys,users,roles,rolebindings,clusterroles,clusterrolebindings
+--format yaml > backup/system-rbac.yaml
+{{< /code >}}
+   
+5. Back up your entity resources (if desired).
+     
+   {{< code shell >}}
+sensuctl dump entities \
+--format yaml | grep -v "^\s*namespace:" > backup/inventory.yaml
+{{< /code >}}
+
+## Restore resources from backup
+
+When you are ready to restore your backed-up resources, use [`sensuctl create`][1].
+
+To restore everything you backed up all at once, run:
+
+{{< code shell >}}
+sensuctl create -r -f backup/
+{{< /code >}}
+
+To restore a subset of your backups (in this example, your pipeline resources), run:
+
+{{< code shell >}}
+sensuctl create -f backup/pipelines.yaml
+{{< /code >}}
+
+{{% notice note %}}
+**NOTE**: To restore `users` resources from `sensuctl dump` output, you must add a `password_hash` or `password` to your `sensuctl create` command.
```

Comment:
To me this note is indicative that we should dump apikeys and users to separate files in dump instructions above, because people won't be able to restore them without modification, if at all -- in the case of API keys, they can't be created from a dump.